dobson/main.py

```diff
@@ -65,18 +67,38 @@ def respond_help(slack_client, channel):
     )
 
 
-def respond_users(slack_client, channel):
+def respond_users(slack_client, channel, list_all=False):
     """Respond with a list of users who are currently at ozone"""
     users = [device.user for device in utils.get_devices()]
+    unknown = list(utils.get_unknown_mac_addresses())
 
-    if len(users) > 2:
-        reply = '{}, and {} are at ozone'.format(', '.join(users[:-1]), users[-1])
-    elif len(users) == 2:
-        reply = '{} and {} are at ozone'.format(users[0], users[1])
+    # generic reply containing who is at ozone, e.g. "A, B and C are at ozone"
+    reply = list_to_str(users)
+    if len(users) == 0:
+        reply = 'Nobody is at ozone.'
     elif len(users) == 1:
-        reply = '{} is at ozone'.format(users[0])
+        reply += ' is at ozone.'
     else:
-        reply = 'Nobody is at ozone'
+        reply += ' are at ozone.'
```

Comment:
The period here conflicts with the comma/period added below, so I'd just get rid of this period/comma so the one below can be used.